scripting/nexus-script-example/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'groovy'
```

Comment:
I used the gradle buildInit plugin to create the basic structure, so I'm not too worried about it